test/test_jit.py

```diff
@@ -11364,6 +11363,21 @@ def split_two(tensor):
         y = torch.randn(3, 6)
         self.checkScript(split_two, [(x + y)])
 
+    def test_python_op_name(self):
+        import random
+
+        with self.assertRaisesRegex(RuntimeError, "randint"):
+            @torch.jit.script
+            def fn():
+                return random.randint()
+
+    def test_type_suggestion(self):
+        with self.assertRaisesRegex(RuntimeError, "did you mean ('float')?"):
+            @torch.jit.script
+            def fn(x):
+                # type: (double) -> int  # noqa
```

Comment:
Where did the double thing come up ?

Reply:
I messed it up once, it's not really essential since `float` is the Python type for floating point numbers but if someone hasn't used mypy much before it might be helpful

Reply:
If it's very rare / not intrinsic to pytorch i would say leave it off but it's up to you

Reply:
Removed it, something like this is more mypy's job than ours